src/pybind/mgr/cephadm/tests/test_scheduling.py

```diff
@@ -182,33 +192,34 @@ def run_scheduler_test(results, mk_spec, get_hosts_func, get_daemons_func, key_e
     (k("123 123 * *"), exactly('1', '2', '3')),
 ]
 
+
 @pytest.mark.parametrize("spec_section_key,spec_section",
-    [
-        ('h', 'hosts'),
-        ('l', 'label'),
-        ('p', 'host_pattern'),
-    ])
+                         [
+                             ('h', 'hosts'),
```

Comment:
why?

Reply:
The format will be kept if it looks like
```
@pytest.mark.parametrize("spec_section_key,spec_section", [
    ('h', 'hosts'),
    ('l', 'label'),
    ('p', 'host_pattern'),
])
```